apps/web/pages/event-types/index.tsx

```diff
@@ -128,17 +125,11 @@ const Item = ({ type, group, readOnly }: { type: EventType; group: EventTypeGrou
           data-testid={"event-type-title-" + type.id}>
           {type.title}
         </span>
-        {group.profile.slug ? (
```

Comment:
this is not from me :o

Reply:
before we merge can we undo this? i dont know where this came from @zomars  @roae